juan-aristizabal/tarea3/spongebob-meme.js

```diff
@@ -0,0 +1,13 @@
+function spongeMeme(sentence) {
+  let palabra = sentence.split("");
+  let nuevaPalabra = "";
+
+  for (let i = 0; i < palabra.length; i++) {
```

Comment:
@santiagoGuillenGlob , use modern approaches for array operations like forEach, map, reduce.